src/stepFunctions/commands/visualizeStateMachine/getStateMachineDefinitionFromCfnTemplate.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs'
+import { getLogger, Logger } from '../../../shared/logger'
+
+/**
+ * @param {string} uniqueIdentifier - unique identifier of state machine
+ * @param {string} templatePath - path for the template.json file
+ *
+ * @returns the escaped ASL Json definition string of the state machine construct
+ */
+export function getStateMachineDefinitionFromCfnTemplate(uniqueIdentifier: string, templatePath: string): string {
+    const logger: Logger = getLogger()
+    try {
+        const data = fs.readFileSync(templatePath, 'utf8')
+        const jsonObj = JSON.parse(data)
+        const resources = jsonObj?.Resources
+        if (!resources) {
+            return ''
+        }
+        let key: string
+
+        const matchingKeyList: string[] = []
+        for (const key of Object.keys(resources)) {
```

Comment:
Rename `key` so it doesn't conflict with the above `let`.

Reply:
Variable is shadowed so there won't be a conflict, though shadowing should definitely be avoided.

Reply:
Updated to change the outer-scoped key variable